google/cloud/privateca/doc/main.dox

```diff
@@ -2,8 +2,11 @@
 
 @mainpage Certificate Authority API C++ Client Library
 
-An idiomatic C++ client library for the [Certificate Authority API][cloud-service-docs], a service
-to The Certificate Authority Service API is a highly-available, scalable service that enables you to simplify and automate the management of private certificate authorities (CAs) while staying in control of your private keys.
+An idiomatic C++ client library for the
+[Certificate Authority API][cloud-service-docs], a service
```

Comment:
This does not read right.  Maybe:

```
... [Certificate Authority API][cloud-service-docs].
Automate the management of private certificate authorities while staying in control of your private keys.
```

Maybe the same change in the README file too?